src/renderer/components/NodeSelectorPanel/NodeSelectorPanel.tsx

```diff
@@ -262,31 +263,32 @@ const NodeSelector = memo(({ schemata, height }: NodeSelectorProps) => {
                     </TabPanels>
                 </Tabs>
             </Box>
-            {showCollapseButtons && (
-                <Button
-                    _hover={{
-                        bg: useColorModeValue('gray.400', 'gray.600'),
-                    }}
-                    aria-label="collapse"
-                    bg={useColorModeValue('gray.300', 'gray.700')}
-                    borderRadius={0}
-                    borderRightRadius="lg"
-                    h="100px"
-                    left={-2}
-                    position="relative"
-                    size="none"
-                    w="0.5rem"
-                    onClick={() => setCollapsed(!collapsed)}
+            <Button
+                _hover={{
+                    bg: useColorModeValue('gray.400', 'gray.600'),
+                    opacity: 1,
+                }}
+                aria-label="collapse"
+                bg={useColorModeValue('gray.300', 'gray.700')}
+                borderRadius={0}
+                borderRightRadius="xl"
+                h="200px"
```

Comment:
I think that's too much. IMO, the problem was that the button was too thin, and you fixed that. `h=100px` is big enough IMO.

![image](https://user-images.githubusercontent.com/20878432/172338316-bb5e919f-b6e5-42b6-bb28-b97a3274d8c5.png)